src/components/pages/NewProject/components/Location.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function Location(props) {
+    return (
+        <div className="col">
+            <label>Location</label>
+          <input
+            required
+            name="location"
+            type="text"
+            className="form-control"
+            placeholder="Location (select on map)"
+            ref={props.register}
+          />
+        </div>
```

Comment:
Replace this with the interactive Google map (as on the New Risk page component)

Reply:
invest require